packages/core/src/bag3d/core/assets/export/archive.py

```diff
@@ -92,6 +92,8 @@ def geopackage_nl(context):
         context.resources.gdal.execute("ogrinfo", cmd)
 
     path_nl_zip = path_nl.with_suffix(".gpkg.zip")
+     # Remove existing
+    path_nl_zip.unlink(missing_ok=True)
```

Comment:
I had to remove existing zip files, because the tests were not running for a second time (the were not overwriting the file)